app/models/user.rb

```diff
@@ -6,4 +6,5 @@ class User < ApplicationRecord
   include DeviseTokenAuth::Concerns::User
   validates :first_name, :last_name, :locale, presence: true
   has_many :rents, dependent: :destroy
+  has_many :book_suggestions, dependent: :destroy
```

Comment:
I am not sure `:destroy` option is the right choice here. Because a book suggestion could not have an user associated, could it?